cmd/dm-tracer/main_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+// Reference: https://dzone.com/articles/measuring-integration-test-coverage-rate-in-pouchc
+
+import (
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestRunMain(t *testing.T) {
+	var args []string
+	for _, arg := range os.Args {
+		switch {
+		case arg == "DEVEL":
+		case strings.HasPrefix(arg, "-test."):
```

Comment:
I have never understood this logic, a `-test.` string prefix?

Reply:
The parameter prefixed with -test is processed by go test and auto added coverage file

Reply:
ok I known